### PR TITLE
Fix TSC

### PR DESCRIPTION
**Pull request by dajinchu**
- Get TSC script to work and not blow up on the decorators error
- Make eslint only lint `.ts` and `.tsx`
- GitHub action prettier check everything in the branch

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 on: pull_request
 
 jobs:
-  lint:
-    name: lint
+  setup:
+    name: setup
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```
Comment (dajinchu) [inline .github/workflows/lint.yml:10]: This is interesting, it makes it so all three commands are run, even if one of them fails. That way we always get the full lint/prettier/eslint output

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "dev": "concurrently -n \"server,app,proxy\" \"yarn workspace @template/server dev\" \"yarn workspace @template/app dev\" \"yarn dev:proxy\"",
     "dev:proxy": "node infrastructure/dev/devProxy.js",
     "tsc": "yarn workspaces run tsc",
-    "lint": "eslint 'packages/**/*.ts*' --ignore-path .gitignore --fix",
```
Comment (dajinchu) [inline package.json:11]: No more autofix. If you want to autofix, you must `yarn lint --fix`

File: packages/common/index.ts [hunk 3]
```diff
@@ -276,13 +276,13 @@ export type GetQuestionResponse = Question;
...
-  text: string;
+  text!: string;
 
   @IsEnum(QuestionType)
-  questionType: QuestionType;
+  questionType!: QuestionType;
 
   @IsInt()
-  queueId: number;
+  queueId!: number;
```
Comment (liustanley) [inline packages/common/index.ts:285]: Just curious, what is the ! for?

Comment (NEUDitao) [reply packages/common/index.ts:285]: tells typescript to not freak out that the value might be null able, cause the value is never nullable

**Pull request merged**